distributed/deploy/tests/test_old_ssh.py

```diff
@@ -28,3 +28,25 @@ def test_cluster(loop):
             while len(e.ncores()) != 3:
                 sleep(0.01)
                 assert time() < start + 5
+
+
+def test_nprocs_renamed_to_num_workers():
+    with pytest.warns(FutureWarning, match="renamed to num_workers"):
+        with SSHCluster(
+            scheduler_addr="127.0.0.1",
+            scheduler_port=8687,
+            worker_addrs=["127.0.0.1", "127.0.0.1"],
+            nprocs=2,
+        ) as c:
+            assert c.num_workers == 2
+
+
+def test_num_workers_with_nprocs_is_an_error():
+    with pytest.raises(ValueError, match="Both nprocs and num_workers"):
+        SSHCluster(
+            scheduler_addr="127.0.0.1",
+            scheduler_port=8687,
+            worker_addrs=(),
+            nprocs=2,
+            num_workers=2,
+        )
```

Comment:
This test is failing, perhaps because I removed the `async` bits here? I'll try in my Docker container to reproduce the failures and fix it.